src/routes/template.rs

```diff
@@ -140,15 +177,32 @@ async fn create(
         is_gs_uris_for_wdls_enabled()?;
     }
 
-    // Start by validating the WDLs
-    validate_wdl(&client, &new_template.test_wdl, "test", &new_template.name).await?;
-    validate_wdl(&client, &new_template.eval_wdl, "eval", &new_template.name).await?;
+    let conn = pool.get().expect("Failed to get DB connection from pool");
+
+    // Store and validate the WDLs
+    let test_wdl_location = store_wdl(&client, &conn, &new_template.test_wdl, WdlType::Test).await?;
```

Comment:
If the WDLs don't validate I feel like we shouldn't store them at all and should just return an error.

What do you think?

Reply:
That makes sense to me.  I have a fix for that now